### PR TITLE
Solution for challenge 1 and 2

### PR DESCRIPTION
**Pull request by saurabhcr007**
Solution by [Saurabh Chaudhary](https://github.com/saurabhcr007/CProject/blob/main/challenge-Solution-1.c) for Challenge-1
Solution by [Saurabh Chaudhary](https://github.com/saurabhcr007/CProject/blob/main/challenge-Solution-2.c) for Challenge-2

### CODE REVIEW

File: new READme.md
Comment (Bharathgopal) [inline new READme.md:1]: Move thit content to README.md file.
Only one file needs to be edited

**Review by Bharathgopal — COMMENTED** [review]
Please do the requested changes before accepting the solution

**Comment by saurabhcr007** [comment]
solutions are moved to README.md file.

**Comment by saurabhcr007** [comment]
solutions are moved to README.md file.

**Comment by Bharathgopal** [comment]
Welcome to the contributors group

**Pull request merged**